also auto-detect MUMPS when using MKL instead of openblas
Author: Miguel Dias Costa (National University of Singapore)
--- setup.py.orig	2019-11-08 16:05:34.300803932 +0800
+++ setup.py	2019-11-08 16:06:04.652430139 +0800
@@ -388,6 +388,8 @@
         # 'openblas' provides Lapack and BLAS symbols
         ['zmumps', 'mumps_common', 'metis', 'esmumps', 'scotch',
          'scotcherr', 'mpiseq', 'openblas'],
+        ['zmumps', 'mumps_common', 'metis', 'esmumps', 'scotch',
+         'scotcherr', 'mpiseq', 'mkl_intel_lp64', 'mkl_sequential', 'mkl_core'],
     ]
     common_libs = ['pord', 'gfortran']
 
